packages/core/test/components/Modal.test.tsx

```diff
@@ -219,6 +219,25 @@ describe('<Modal />', () => {
     expect(titleWrapper.prop('children')).toBe('Title wave');
   });
 
+  it('render a large titel when prop large is true', () => {
```

Comment:
`title`

Reply:
did you want to add a test for `large` or `compact`? please add a test for `compact` to reflect the changes in the PR

Reply:
Will do ! Thanks @stefhatcher